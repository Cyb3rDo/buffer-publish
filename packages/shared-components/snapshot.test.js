// use story.js files as snapshots
import initStoryshots, {
  shallowSnapshot,
} from '@storybook/addon-storyshots';

initStoryshots({
  suit: 'Snapshots',
  test: shallowSnapshot,
});
