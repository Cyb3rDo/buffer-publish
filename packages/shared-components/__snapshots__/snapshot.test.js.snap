// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Example default 1`] = `
<span>
  <div>
    Example
  </div>
</span>
`;
